tensorflow/tools/pip_package/build_pip_package.sh

```diff
@@ -107,7 +107,8 @@ function main() {
   mkdir -p ${TMPDIR}/third_party
   pushd ${RUNFILES%org_tensorflow}
   for header in $(find protobuf -name \*.h); do
-    cp --parents "$header" ${TMPDIR}/google;
+    mkdir -p "${TMPDIR}/google/$(dirname ${header})"
+    cp "$header" "${TMPDIR}/google/$(dirname ${header})/"
```

Comment:
Not very important, but a minor nit.
Do we need to call "dirname" here?
The destination file can be specified, I do not see an issue with that in this line.